Formula/t/tmux-sessionizer.rb

```diff
@@ -0,0 +1,48 @@
+class TmuxSessionizer < Formula
+  desc "Tool for opening git repositories as tmux sessions"
+  homepage "https://github.com/jrmoulton/tmux-sessionizer/"
+  url "https://github.com/jrmoulton/tmux-sessionizer/archive/refs/tags/v0.4.2.tar.gz"
+  sha256 "0f9369a045ebe181202fcf5f292bbdd836f25b47ca9da1702351a725693631f5"
+  license "MIT"
+
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on "libgit2@1.7"
```

Comment:
I saw this pr https://github.com/jrmoulton/tmux-sessionizer/pull/104 has updated to use libgit2 1.8, any timeline to release that change?

Reply:
I asked for a new release a while back here: https://github.com/jrmoulton/tmux-sessionizer/issues/123
The maintainer doesn't seem too active lately, aside from merging one PR recently